src/quo2/components/wallet/keypair/style.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.wallet.keypair.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [selected? customization-color theme]
+  {:width          335
```

Comment:
I think we should use padding instead of hardcoding a width.

Reply:
I agree but for the outer container view imo best to use a container-style prop to handle external padding/margin 👍 and just make this flex

Reply:
@ibrkhalil I think I should use flex. Not width or padding. Padding would still be hardcoding.